src/mpris.rs

```diff
@@ -118,7 +115,13 @@ fn get_metadata(playable: Option<Playable>) -> Metadata {
     );
     hm.insert(
         "xesam:url".to_string(),
-        Variant(Box::new(playable.map(|t| t.uri()).unwrap_or_default())),
+        Variant(Box::new(
+            playable
+                .map(|t| {
+                    t.share_url().unwrap_or_else(|| "0".to_string())
```

Comment:
This will set `xesam:url` to `0` if no URL is set. Is this according to specs?

Reply:
I'm not sure, how would I check that?
I haven't ever seen Spotify not returning an url

Reply:
Perhaps it would be better to just set `xesam:url` to an empty string if no url is found?

Reply:
I just checked, and when getting metadata for podcasts, `mpris:artUrl` and `xesam:artist` is just empty, no "0" or anything, so `xesam:url` should probably do the same

Reply:
Empty string sounds more appropriate than `"0"`.

Reply:
Yeah, I'll change it.